_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp

```diff
@@ -251,6 +249,38 @@ mfxStatus SetRateControl(
     return MFX_ERR_NONE;
 }
 
+mfxStatus SetBrcResetRequired(
+    VADisplay             vaDisplay,
+    VAContextID           vaContextEncode,
+    VABufferID &          rateParamBuf_id,
+    bool                  isBrcResetRequired)
+{
+    VAStatus vaSts;
+    VAEncMiscParameterBuffer *misc_param;
+    VAEncMiscParameterRateControl *rate_param;
+
+    mfxStatus mfxSts = CheckAndDestroyVAbuffer(vaDisplay, rateParamBuf_id);
```

Comment:
please, don't do it. You need to accept id of buffer and map / unmap it (and change the desired value between)

Reply:
Thanks, got my mistake